obfx_modules/mystock-import/js/src/components/InsertImage.js

```diff
@@ -0,0 +1,13 @@
+const { createBlock } = wp.blocks;
```

Comment:
`wp.blocks` has been deprecated so it won't work in newer versions soon. It has been changed to `wp.blockEditor`.

You can write it like:

```const { createBlock } = wp.blockEditor || wp.editor;```

Reply:
@HardeepAsrani wp.blockEditor does not have any createBlock function. How do you create a block using it?

Reply:
@HardeepAsrani it seems that wp.blockEditor does not have a createBlock method. I'm getting this:

TypeError: createBlock is not a function
    at PhotoList.InsertImage (InsertImage.js:22)
    at eval (Photo.js:94)

Reply:
Yes, sorry, not for this. Wrong one. It's with the wp.editor ones, not wp.blocks. My mistake.